### DISCUSSION TITLE
Should we release 0.71.5?

### DISCUSSION THREAD

**Discussion by kelset** [reactwg/react-native-releases#59] [Patches]
## Released. Please refer to #60 

Current Release: [0.71.4](https://github.com/facebook/react-native/releases/tag/v0.71.4)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1. https://github.com/facebook/react-native/commit/1d22e291462ac452f2bb6b1b6af11986944ec54a (already on branch)
2. https://github.com/facebook/react-native/commit/8d8b44a5e33d98f3f80aaa23f3d605717d96cd0f
3. https://github.com/facebook/react-native/commit/cbaf4c8da295cb9235a978d39a0ae8bca314c6b2#r103992924 <-- Fix as local commit and bump RNGP
4. https://github.com/facebook/react-native/commit/36cc71ab36aac5e5a78f2fbae44583d1df9c3cef
5. https://github.com/facebook/react-native/commit/c0abff11b66d9ec3a8e1d09333a3fb6c05678bed
6. Bump Metro/CLI
  * https://github.com/react-native-community/cli/pull/1863
  * https://github.com/facebook/metro/pull/949
  -> version: 
    - @react-native-community/cli-doctor@10.2.2
    - @react-native-community/cli-plugin-metro@10.2.2
    - @react-native-community/cli@10.2.2
7. https://github.com/facebook/react-native/commit/9aac13d4dc95925b57f03e7964fc7add6834e518

#### Local commits to backport to main

1.

**Comment by cipolleschi** [comment]
Remove Ruby requirements:
- https://github.com/facebook/react-native/commit/e57b6d11fa39aa96d92a7a6b8c91429974bef81c

**Comment by kelset** [comment]
https://github.com/facebook/react-native/pull/36421

**Comment by kelset** [comment]
probably fix needed for RNGP: https://github.com/facebook/react-native/commit/cbaf4c8da295cb9235a978d39a0ae8bca314c6b2

(we can do as local commit and bump RNGP)

**Reply by cortinico** [reply]
This has already been picked on the 0.71-stable branch (I see it in the history picked as `cbaf4c8da295cb9235a978d39a0ae8bca314c6b2`)

**Reply by kelset** [reply]
I mean the comment in the commit ;) we need to fix that

**Comment by kelset** [comment]
samsung lists + textinput issue: [this week a fix should land](https://github.com/facebook/react-native/issues/35936#issuecomment-1464830542)

> The larger change to minimize the number of spans emitted by React Native is currently being validated and is targeting next week to hit the main branch.

We should wait for it before doing a new patch.

**Comment by lachenmayer** [comment]
Hey there, it would be amazing if the [Prevent native blob resource from being de-allocated prematurely](https://github.com/facebook/react-native/pull/31392) commit could be released at some point. We still have to patch RN just for this (see [my comment](https://github.com/facebook/react-native/pull/31392#issuecomment-1224635928) on the PR). Thank you!

https://github.com/facebook/react-native/commit/36cc71ab36aac5e5a78f2fbae44583d1df9c3cef

**Reply by fortmarek** [reply]
We will try cherry-picking this and the commit will be included in the next patch if there are no merge conflicts 👍

**Comment by thymikee** [comment]
FYI, we ported a fix for listing iOS simulators and devices through CLI: https://github.com/react-native-community/cli/pull/1863
This could be pulled to the next 0.71.x

**Reply by kelset** [reply]
this would mean bump to these versions right?
```
 - @react-native-community/cli-doctor@10.2.1
 - @react-native-community/cli-platform-ios@10.2.1
 - @react-native-community/cli@10.2.1
```

**Reply by thymikee** [reply]
yuup

**Comment by khayym** [comment]
Lineheight no working correctly 

![Simulator Screen Shot - iPad Pro (12 9-inch) (6th generation) - 2023-03-16 at 18 05 46](https://user-images.githubusercontent.com/68564696/225642672-a4248419-69b2-464f-8db7-cb17c1b5976a.png)

**Reply by fortmarek** [reply]
Hey 👋 

Please create a [new issue](https://github.com/facebook/react-native/issues/new/choose) in the `react-native` repository with a reproducible sample instead 🙏  This discussion is not meant for reporting bugs unless they are major release issues.

**Comment by lunaleaps** [comment]
In response to https://github.com/facebook/react-native/issues/36229, want to pick https://github.com/facebook/react-native/commit/c0abff11b66d9ec3a8e1d09333a3fb6c05678bed to fix the breaking change

Here is the corresponding documentation PR: https://github.com/facebook/react-native-website/pull/3624

**Comment by robhogan** [comment]
A fix is in the pipeline for the significant Windows/Linux Metro regression reported in https://github.com/facebook/react-native/issues/36387. We'll be releasing [Metro 0.73.9](https://github.com/facebook/metro/pull/949) as a hotfix and then will need a CLI hotfix bump too - IMO this will be an important one to get into 0.71.5.

**Reply by kelset** [reply]
thanks for the headsup! we'll make sure we put it in 👍

**Reply by robhogan** [reply]
Hotfix released in Metro 0.73.9. CLI PR: https://github.com/react-native-community/cli/pull/1886

**Reply by robhogan** [reply]
Metro 0.73.9 included in CLI 10.2.2. Updating 0.71-stable: https://github.com/facebook/react-native/pull/36678

**Reply by kelset** [reply]
sorry Rob looks like we worked on this at the same time: https://github.com/facebook/react-native/commit/4c3bc24893b2dc7495a2e65ee8f1c6408cc31ad5 I'll close off your PR

**Comment by Rag0n** [comment]
Hey, it would be nice to pick crash fix https://github.com/facebook/react-native/commit/9aac13d4dc95925b57f03e7964fc7add6834e518

**Comment by Saadnajmi** [comment]
Related to https://github.com/reactwg/react-native-releases/discussions/59#discussioncomment-5255267
Could we also pick https://github.com/facebook/react-native/commit/0f56cee8e1fca9575e83f439274b83e01bdd98e2 to get the latest cocoa pods version?

**Reply by kelset** [reply]
I didnt' want to enforce the bump of cocoapods in a patch release since it might disrupt some folks' codebases - isn't it something you can do downstream? 🤔

**Reply by Saadnajmi** [reply]
Yep, planning to do it downstream anyway.
